binsync/extras/server/__init__.py

```diff
@@ -1,3 +1,28 @@
-def start_server():
-    print("Starting BinSync server...")
-    # TODO: implement server
\ No newline at end of file
+from flask import Flask
+from threading import Lock
+app = Flask(__name__)
+
+user_count_lock = Lock()
+user_count = 0
+@app.route('/connect')
+def handle_connection():
+    global user_count
+    with user_count_lock:
+        user_count += 1
+        print(user_count)
+    return 'You are connected!'
+
+@app.route("/disconnect")
+def handle_disconnection():
+    global user_count
+    with user_count_lock:
+        user_count -= 1
+        print(user_count)
+    return 'You have disconnected!'
+
+# main driver function
+def start_server(port=7962):
+    print("starting server!")
```

Comment:
In the future, avoid all `print`. Try to never have these in code. We always want a way to programmatically disable output that can clog stdout.